webui/react/src/pages/ExperimentDetails/CodeViewer/IpynbRenderer.tsx

```diff
@@ -1,7 +1,6 @@
+import NotebookJS from 'notebook';
```

Comment:
question: not part of this PR, but question for @EmilyBonar (I think): there must be a reason but curious as to why `notebook.js` can't just be in the `package.json` and call it a day

Reply:
We vendored `notebook.js` because the bundler includes `jsdom` in our bundle, so the vendored version now assumes it's running in a browser.